src/application/services/useAuth.ts

```diff
@@ -59,3 +66,19 @@ export default function useOAuth(): UseOAuthComposableState {
     showGoogleAuthPopup,
   };
 }
+
+/**
+ * Methods for working with Auth tokens
+ */
+export function useAuth(): UseAuthComposableState {
+  /**
+   * Deletes the refresh token in the local storage
+   */
+  async function logoutTheUser(): Promise<void> {
```

Comment:
I guess we can just add this method to existing composable, no need to create another one. Just rename useOAuth to useAuth